clients/pkg/promtail/targets/cloudflare/target.go

```diff
@@ -151,7 +151,10 @@ func (t *Target) pull(ctx context.Context, start, end time.Time) error {
 
 	for backoff.Ongoing() {
 		it, err = t.client.LogpullReceived(ctx, start, end)
-		if err != nil {
+		if err != nil && err.Error() == "HTTP status 400: bad query: error parsing time: invalid time range: too early: logs older than 168h0m0s are not available" {
```

Comment:
Hm, could we make this a constant and maybe match it a little bit better. Is there a chance the client exposes the HTTP status?

Reply:
Agreed, this is a bit flimsy

Reply:
Sure I'll make a constant, I'm not sure the best match maybe just for the invalid time range: too early: logs older than 168h0m0s part... Any suggestions. As for status there are a ton of valid retry 400 error's but only this one should be fully ignored as far as I can see from the cloudflare docs.

But will add the 400 portion.